packages/core/src/cart/cart-request-body.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * An object that contains the information required for creating cart.
+ */
+export default interface CartRequestBody {
```

Comment:
Can we call this `BuyNowCartRequestBody`?

Reply:
Yes 👍